.github/workflows/main.yml

```
@@ -140,6 +184,45 @@ jobs:
       env:
         BUTLER_CREDENTIALS: ${{ secrets.ITCH_IO_API_KEY }}
         CHANNEL: HTML
-        ITCH_GAME: godot-ci
+        ITCH_GAME: ${{ env.PROJECT_NAME }}
         ITCH_USER: gmcunimelb
         PACKAGE: ${{ env.PROJECT_NAME }}_web.zip
+  
+  Steam-deploy:
+    needs: [Builds]
+    runs-on: ubuntu-latest
+    steps:
```

Ideally we only do this, also find a way to only run this when we have `STEAM_APPID` defined.
```suggestion
    Steam-deploy:
    needs: [Builds]
    runs-on: ubuntu-latest
    if: github.event_name == 'workflow_dispatch' || github.event_name == 'pull_request' || github.event_name == 'push'
    steps:
```